src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -1074,6 +1074,9 @@ Map options:
   --device <device path>  Optional mapping unique identifier
   --exclusive             Forbid writes by other clients
   --read-only             Map read-only
+  --wait-online           Wait for the disk to become available.
+  --wait-online-timeout   Specifies how long to wait for the disk to become
+                          available (seconds). Default: 15
```

Comment:
`DEFAULT_IMAGE_MAP_TIMEOUT` is defined to 20.

Reply:
Whops, thanks.